ui/ui-frontend/projects/collect/src/app/collect/transactions/transaction-list/transaction-list.component.ts

```diff
@@ -171,13 +196,20 @@ export class TransactionListComponent extends InfiniteScrollTable<Transaction> i
   }
 
   transactionIsEditable(transaction: Transaction): boolean {
+    if (this.isAutomaticIngest && [TransactionStatus.READY, TransactionStatus.VALIDATED].indexOf(transaction.status) !== -1) {
```

Comment:
```suggestion
    if (this.isAutomaticIngest && [TransactionStatus.READY, TransactionStatus.VALIDATED].includes(transaction.status)) {
```
D'ailleurs, on peut remplacer les autres `indexOf() !== -1` dans le fichier

Reply:
Il reste juste 1 indexOf dans le return de transactionIsEditable